Doc/library/dis.rst

```diff
@@ -1365,6 +1428,11 @@ Opcode collections
 These collections are provided for automatic introspection of bytecode
 instructions:
 
+   .. versionchanged:: 3.12
+      The collections now contain virtual opcodes as well. These are
+      opcodes in the range between ``MIN_VIRTUAL_OPCODE`` and
+      ``MAX_VIRTUAL_OPCODE``.
```

Comment:
I can call it MAX_OPCODE. It's used in some assertion about an opcode arg being within range. It could be useful for someone wanting to iterate over the valid range (currently 255 shows up as MAX_OPCODE in a few places).

Reply:
`MAX_VIRTUAL_OPCODE` is fine. I just wondered why it was needed.